README.md

```diff
@@ -6,19 +6,22 @@ One edge case to keep in mind is when you want an image to size to its container
 ## Proposed Solution
 Add an intrinsic-size attribute: `<img intrinsicsize="400x300" style="width: 100%">`.
 
-This attribute tells the browser to ignore the actual intrinsic size of the image and pretend it’s the size specified in the attribute. Specifically, the image would raster at these dimensions and [naturalWidth/naturalHeight](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-naturalwidth) on images would return the values specified in this attribute. On video elements, the video would raster at this size and the [videoWidth/videoHeight](https://html.spec.whatwg.org/multipage/media.html#dom-video-videowidth) on the video would return the intrinsicsize values.
+This attribute tells the browser to ignore the actual intrinsic size of the image and pretend it’s the size specified in the attribute. Specifically, the image would raster at these dimensions and [naturalWidth/naturalHeight](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-naturalwidth) on images would return the values specified in this attribute. On video elements, the video would raster at this size and the [videoWidth/videoHeight](https://html.spec.whatwg.org/multipage/media.html#dom-video-videowidth) on the video would return the value of the 'intrinsicsize' attribute.
 
 All other image size operations behave the same. So, for example:
-* If no width/height are otherwise set, then the image still be the size specified by intrinsicsize
+* If no width/height are otherwise set, then the image still be the size specified by 'intrinsicsize'
 * If the width is set on the image, then it would set the height to maintain the aspect ratio defined in this property.
-* If width and height are set on the image, then this attribute’s value is ignored.
+* If width and height are set on the image, then this attribute’s value is ignored (however, it will still override the image's intrinsic size).
```

Comment:
Would it be more precise to say something like the following or am I misunderstanding?

If width and height are set on the image, then this attribute’s value only affects the values of naturalWidth/naturalHeight, but not the rendered size of the image.

Reply:
Yes!